ietf/iesg/tests.py

```diff
@@ -182,6 +185,1578 @@ def test_ietf_activity(self):
         r = self.client.get(url)
         self.assertEqual(r.status_code, 200)
 
+    def test_working_groups(self):
+        # Clean away the wasted built-for-every-test noise
+        Group.objects.filter(type__in=["wg", "area"]).delete()
+
+        (
+            area_summary,
+            area_totals,
+            ad_summary,
+            noad_summary,
+            ad_totals,
+            noad_totals,
+            totals,
+            wg_summary,
+        ) = get_wg_dashboard_info()
+        self.assertEqual(area_summary, [])
+        self.assertEqual(
+            area_totals, {"group_count": 0, "doc_count": 0, "page_count": 0}
+        )
+        self.assertEqual(ad_summary, [])
+        self.assertEqual(noad_summary, [])
+        self.assertEqual(
+            ad_totals,
+            {
+                "ad_group_count": 0,
+                "doc_group_count": 0,
+                "doc_count": 0,
+                "page_count": 0,
+            },
+        )
+        self.assertEqual(
+            noad_totals,
+            {
+                "ad_group_count": 0,
+                "doc_group_count": 0,
+                "doc_count": 0,
+                "page_count": 0,
+            },
+        )
+        self.assertEqual(
+            totals,
+            {
+                "group_count": 0,
+                "doc_count": 0,
+                "page_count": 0,
+                "groups_with_docs_count": 0,
+            },
+        )
+        self.assertEqual(wg_summary, [])
+
+        # Areas with WGs following the "normal" pattern
+        areas = {}
+        for area_acronym in ["red", "orange", "yellow", "green", "blue", "violet"]:
+            areas[area_acronym] = GroupFactory(type_id="area", acronym=area_acronym)
+        for ad, area, wgs in [
```

Comment:
I think a comment - at least a warning - that there's method to the naming madness would be appropriate here. This should warn at least that there's significance to the first letter of the wg acronyms.